.changeset/great-emus-confess.md

```diff
@@ -0,0 +1,50 @@
+---
+"next-blitz-auth": major
+"toolkit-app": major
+"toolkit-app-passport": major
+"web": major
+"test-auth": major
+"test-full-auth-with-rpc": major
+"test-get-initial-props": major
+"test-middleware": major
+"test-app-dir": major
+"test-no-suspense": major
+"test-qm": major
+"test-react-query-utils": major
+"test-rpc": major
+"test-rpc-path-root": major
+"test-trailing-slash": major
+"test-utils": major
+"blitz": major
+"@blitzjs/auth": major
+"@blitzjs/next": major
+"@blitzjs/rpc": major
+"@blitzjs/codemod": major
+"@blitzjs/config": major
+"@blitzjs/generator": major
+"template": major
+"@blitzjs/recipe-base-web": major
+"@blitzjs/recipe-bulma": major
+"@blitzjs/recipe-bumbag-ui": major
+"@blitzjs/recipe-chakra-ui": major
+"@blitzjs/recipe-emotion": major
+"@blitzjs/recipe-gh-action-yarn-mariadb": major
+"@blitzjs/recipe-gh-action-yarn-postgres": major
+"@blitzjs/recipe-ghost": major
+"@blitzjs/recipe-graphql": major
+"@blitzjs/recipe-logrocket": major
+"@blitzjs/recipe-material-ui": major
+"@blitzjs/recipe-next-ui": major
+"@blitzjs/recipe-passenger": major
+"@blitzjs/recipe-quirrel": major
+"@blitzjs/recipe-reflexjs": major
+"@blitzjs/recipe-render": major
+"@blitzjs/recipe-secureheaders": major
+"@blitzjs/recipe-stitches": major
+"@blitzjs/recipe-styled-components": major
+"@blitzjs/recipe-tailwind": major
+"@blitzjs/recipe-theme-ui": major
+"@blitzjs/recipe-vanilla-extract": major
+---
+
+I have changed User model in all of Prisma's schemas. I've changed the role key, changing it from a single string to an array of string called "roles". Then, I've changed all of functions where user is created to grant him a default roles array ["user"] instead of a single string "user". I've also changed all of types that where related to User's role and I've changed line 351 of file "blitz/packages/blitz-auth/src/client/index.tsx", to make the condition accept an array of string for roles.
```

Comment:
🔨 this is a user facing changeset that will be present in the github release, so the text here should be on the user facing changes implemented here